recipes-support/curl/curl/0013-Don-t-use-wincrypt.h-on-PharLap.patch

```diff
@@ -1,30 +1,27 @@
-From 9a75327ddce479d76ab0c45d05e62b180f96f28e Mon Sep 17 00:00:00 2001
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
 From: Jacob Champion <jacob.champion@ni.com>
```

Comment:
Could you update the attribution info, since you modified the .patch contents.

Reply:
Amended.